### DISCUSSION TITLE
Composite Layers

### DISCUSSION THREAD

**Discussion by anujdivesh** [TerriaJS/terriajs#5909] [Q&A] [answered]
I was trying to configure composite layers in terriav8 ( "type": "composite"). I get the following error:
![image](https://user-images.githubusercontent.com/82851258/137060302-999ab0e9-3c6f-42f9-8dc8-86eecc172761.png)

Any leads will be much appreciated.

**Comment by nf-s** [comment]
Hi @anujdivesh 

This bug was fixed in terriajs version `8.1.1` - https://github.com/TerriaJS/terriajs/pull/5859

Which version of terriajs are you using?

Cheers,
Nick

**Reply by anujdivesh** [reply]
Hi @nf-s 
I have re-built Terria-Map with the latest terriajs. The issue still persists. If you have any example, would be great.
Also note, my catalog has wms with time-series data.

Thanks again

**Comment by nf-s** [comment, marked as answer]
Hi @anujdivesh 

This example should work fine - http://ci.terria.io/main/#clean&https://gist.githubusercontent.com/steve9164/e3cbc14198be899f885079731c0f6393/raw/437b3f8c11230aca33dc848c9156712e69b32d16/test-composite.json&share=s-rbCmIeTpytOOvGMMdTbl3wTCf42

It uses the following config - with two WMS items

```json
{
  "workbench": [],
  "catalog": [
    {
      "type": "composite",
      "name": "FTTP and HFC NBN Connections - July 2020",
      "members": [
        {
          "name": "FTTP NBN Connections - July 2020",
          "layers": "FTTP",
          "url": "https://programs.communications.gov.au/geoserver/national-broadband-network/wms?request=GetCapabilities",               
          "type": "wms"
        },
        {
          "name": "HFC NBN Connections - July 2020",
          "layers": "HFC",
          "url": "https://programs.communications.gov.au/geoserver/national-broadband-network/wms?request=GetCapabilities",               
          "type": "wms"
        }
      ]
    }
  ],
  "homeCamera": {
    "north": -8,
    "east": 158,
    "south": -45,
    "west": 109
  }
}
```

I have made a pull request to update terriajs to version `8.1.4` in [TerriaMap](https://github.com/TerriaJS/TerriaMap)

https://github.com/TerriaJS/TerriaMap/pull/558/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519

**Comment by anujdivesh** [comment]
Hi @nf-s 
Perfect works like a charm. :+1: 
One quick question, if we can enable time-series slider at the bottom while having composite layers?

Thanks

**Comment by nf-s** [comment]
Hi @anujdivesh 
Sorry time currently isn't supported for `CompositeCatalogItem` - I have made a ticket https://github.com/TerriaJS/terriajs/issues/5932

**Reply by anujdivesh** [reply]
Hi @nf-s :
Thanks so much, will surely keep an eye on this.

**Reply by imakihi** [reply]
It would be super nice if we could use the time slider with a composite catalog item. I have a czml animation file and wanted to use the composite catalog item with 3D tiles, but I couldn't show as animation...
Hiroo
